src/test/java/uk/gov/hmcts/reform/et/syaapi/utils/TestConstants.java

```diff
@@ -8,6 +8,7 @@ public final class TestConstants {
     public static final String DRAFT = "DRAFT";
     public static final String AWAITING_SUBMISSION_TO_HMCTS = "AWAITING_SUBMISSION_TO_HMCTS";
     public static final String SUBMITTED = "SUBMITTED";
+    public static final String TEST_STRING = "TEST";
 
     private TestConstants() {
         // Utility clasa
```

Comment:
// Utility class

Reply:
Let me remove the comment.

Reply:
Cannot remove this as check style was complaining.